providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/FlagdProviderSyncResourcesTest.java

```diff
@@ -0,0 +1,140 @@
+package dev.openfeature.contrib.providers.flagd;
+
+import dev.openfeature.sdk.exceptions.GeneralError;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicLong;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+
+class FlagdProviderSyncResourcesTest {
+    private static final long PERMISSIBLE_EPSILON = 20;
```

Comment:
Perhaps rename this to for the benefit of those of us who haven't been in a calculus class for a decade :sweat_smile: